File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.py
def solution(score):
    answer = []
    result = []
    #평균내서 배열에 담기
    for i in score:
        answer.append((i[0]+i[1])/2)
    #등수 매기기
    s = sorted(answer, reverse=True)
    for i in answer:
        result.append(s.index(i)+1)
    return result

File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.js
function solution(score) {
    var answer = [];
    var s = [];
    //평균값으로 배열에 넣기
    for(var i =0;i<score.length;i++){
        score[i] = (score[i][0]+score[i][1])/2;
    }
    s = [...score].sort((a,b)=>b-a);
    for (let a of score){
        answer.push(s.indexOf(a)+1)
    }
    return answer;
}